src/Kunstmaan/AdminBundle/Twig/MultiDomainAdminTwigExtension.php

```diff
@@ -42,19 +45,18 @@ public function getFunctions()
      *
      * @return string
      */
-    public function renderWidget(\Twig_Environment $env, $route, array $parameters = array())
+    public function renderWidget(Twig_Environment $env, $route, array $parameters = [])
     {
         $template = $env->loadTemplate(
             "@KunstmaanAdmin/MultiDomainAdminTwigExtension/widget.html.twig"
         );
 
         return $template->render(
             array_merge(
```

Comment:
Maybe also use function with namespace is better choice?
[https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/](https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/)

Reply:
I think that's unnecessary in this case as we have explicitly declared which class it is in the use statements at the top.  I think that just applies to functions rather than classes.

Reply:
Just spoke with Ocramius (my friend Marco, the guy whose tweet was in the link you sent me). He always does it the same way as I have, and confirmed it's for core function calls and not classes :-)

Reply:
I don't know if you understood me correctly i check only ```array_merge``` function but git showed huge part of function ;)

Reply:
Ah! Yes I see now. I thought you highlighted the \Twig_Environment! :-P

Reply:
Fixing now! 👍😃

Reply:
:+1: